Description: hd is not supposed to understand option '-C' because it is implied
Author: Michael Meskes <meskes@debian.org>

--- bsdmainutils/usr.bin/hexdump/hexsyntax.c	2009-10-04 02:56:53.000000000 +0200
+++ bsdmainutils/usr.bin/hexdump/hexsyntax.c	2009-10-04 02:56:15.000000000 +0200
@@ -75,6 +75,11 @@
 			add("\"%07.7_ax \" 16/1 \"%3_c \" \"\\n\"");
 			break;
 		case 'C':
+			/* hd is not supposed to understand this option because it is already implied */
+			if ((p = rindex(argv[0], 'h')) != NULL &&
+			    strcmp(p, "hd") == 0)
+				usage();
+				
 			add("\"%08.8_Ax\n\"");
 			add("\"%08.8_ax  \" 8/1 \"%02x \" \"  \" 8/1 \"%02x \" ");
 			add("\"  |\" 16/1 \"%_p\" \"|\\n\"");
